src/Raven.Server/Documents/Handlers/PullReplicationHandler.cs

```diff
@@ -103,6 +113,15 @@ public async Task UnregisterHubAccess()
                 var command = new UnregisterReplicationHubAccessCommand(Database.Name, hub, thumbprint, GetRaftRequestIdFromQuery());
                 var result = await Server.ServerStore.SendToLeaderAsync(command);
                 await WaitForIndexToBeApplied(context, result.Index);
+                
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.OK;
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    writer.WriteStartObject();
+                    writer.WritePropertyName("TaskIndex");
```

Comment:
Can we use nameof? Same name issue as above.

Reply:
done